upgrades/operations.go

```diff
@@ -20,6 +20,7 @@ import (
 var stateUpgradeOperations = func() []Operation {
 	steps := []Operation{
 		upgradeToVersion{version.MustParse("3.3.1"), stateStepsFor331()},
+		upgradeToVersion{version.MustParse("3.3.4"), stateStepsFor334()},
```

Comment:
Uh oh

Version here needs a change

Looks like someone else made the same mistake though previously. We should investigate ConvertApplicationsOfferTokenKeys and fix before merging this

Reply:
Changed the version and updated the tests.